### ISSUE TITLE
`wrapper.js` stability - adding global error handlers 

### ISSUE THREAD

**Issue by NitayRabi** [coreybutler/node-windows#219]
Issue - 
*servicename*.err.log outputs logs indicating `wrapper.js` process/server errors stopping the whole process - e.g 
```
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at _errnoException (util.js:1022:11)
    at TCP.onread (net.js:615:25)
```

To handle these issues i suggest adding - 
```javascript
let server = require('net').createServer().listen();
server.on('error', (err) => {
    launch('warn', err.message);
    server = require('net').createServer().listen();
});
process.on('uncaughtException', (err) => {
    launch('warn', err.message);
});
```

**Comment by NitayRabi** [comment]
#220

**Comment by coreybutler** [comment]
Obviously I haven't been maintain this for awhile. I merged your PR and it's in 1.0.0-beta.1, thank you!
